SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -400,6 +402,11 @@ void USpatialReceiver::HandleActorAuthority(const Worker_AuthorityChangeOp& Op)
 		LoadBalanceEnforcer->AuthorityChanged(Op);
 	}
 
+	if (NetDriver->SpatialDebugger != nullptr)
+	{
+		NetDriver->SpatialDebugger->ActorAuthorityChanged(Op);
```

Comment:
isn't this getting called for components != AUTHORITY_INTENT_COMPONENT_ID?  Internally ActorAuthorityChanged checks() on that condition

Reply:
Yup, you're absolutely right. Clearly I didn't rerun the test after the last code review pass! Changed the check back to an if conditional.